cube_browser/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [[ $(uname) == "Darwin" ]]; then
```

Comment:
This if statement isn't used, so the Windows and Unix build commands are the same. In which case, we can simply make this part of the `meta.yaml` under the `build/script` section. http://conda.pydata.org/docs/building/meta-yaml.html#script